internal.h

```diff
@@ -2403,11 +2390,6 @@ const char *rb_objspace_data_type_name(VALUE obj);
 /* Temporary.  This API will be removed (renamed). */
 VALUE rb_thread_io_blocking_region(rb_blocking_function_t *func, void *data1, int fd);
 
-/* array.c (export) */
-void rb_ary_detransient(VALUE a);
```

Comment:
These functions are used in include/ruby/ruby.h.

Reply:
True.  They are not deleted.  I have deleted those duplicated declarations in internal.h, which are totally out of use.